pkgs/development/tools/build-managers/bazel/default.nix

```diff
@@ -321,10 +322,11 @@ stdenv.mkDerivation rec {
     '';
 
     genericPatches = ''
-      # Substitute python's stub shebang to plain python path. (see TODO add pr URL)
+      # Substitute j2objc and objc wrapper's python shebang to plain python path.
```

Comment:
we should explain why we replace python2 stuff here, but python3 further down

Reply:
I've added a comment explaining this.